src/apps/relay/ns_ioalib_engine_impl.c

```diff
@@ -3655,7 +3655,13 @@ void turn_report_allocation_delete(void *a, SOCKET_TYPE socket_type) {
                                       (unsigned long)(ss->t_peer_sent_packets), (unsigned long)(ss->t_peer_sent_bytes),
                                       true);
           }
-          prom_dec_allocation(socket_type);
+          turn_time_t ct = get_turn_server_time(server) - ss->start_time;
+          uint64_t received_rate_kbps = ss->received_rate / 125;
+          uint64_t sent_rate_kbps = ss->sent_rate / 125;
```

Comment:
What is 125 here?

Reply:
bytes per second to Kilobit per second. I went ahead and defined a const for it.